### DISCUSSION TITLE
Video tutorial of using this library

### DISCUSSION THREAD

**Discussion by samos123** [acheong08/ChatGPT#465] [Show and tell]
https://youtu.be/Im6mh2ZFWx8

**Comment by wzqstudio** [comment]
thks

**Comment by acheong08** [comment]
Thanks! I didn't catch this one. 

It's a bit outdated since Official API got released. Might be useful to update (if possible on YouTube? I don't know)

**Comment by samos123** [comment]
I checked the official OpenAI docs but there is no official API for ChatGPT yet? I have been getting good results with just using the GPT-3 API though

**Reply by acheong08** [reply]
There is. It's secret and undocumented. We brute forced it

**Reply by acheong08** [reply]
`text-chat-davinci-002-20221122`

**Reply by acheong08** [reply]
Check out new readme

**Reply by samos123** [reply]
Lol nice! I wouldn't call that an official API though. It's another unofficial API that can use your OpenAI API key instead. It might be removed it might change at any moment. I'm gonna try it out soon. Thanks for sharing! I might make another video.

**Reply by acheong08** [reply]
According to sources from @transitive-bullshit, it is officially used by VC/YC partners of OpenAI
